sdk/trace/batch_span_processor.go

```diff
@@ -265,7 +266,15 @@ func (bsp *batchSpanProcessor) exportSpans(ctx context.Context) error {
 	}
 
 	if l := len(bsp.batch); l > 0 {
-		global.Debug("exporting spans", "count", len(bsp.batch), "total_dropped", atomic.LoadUint32(&bsp.dropped))
+		global.Debug("exporting spans", "count", len(bsp.batch))
+
+		dropped := atomic.LoadUint32(&bsp.dropped)
+		reportedDropped := atomic.SwapUint32(&bsp.reportedDropped, dropped)
+		if dropped > reportedDropped {
+			droppedThisBatch := dropped - reportedDropped
+			global.Info("dropped spans", "total", dropped, "this_batch", droppedThisBatch)
```

Comment:
Should we not call `global.Warn`?

```suggestion
			global.Warn("dropped spans", "total", dropped, "this_batch", droppedThisBatch)
```